lib/deimos/config/configuration.rb

```diff
@@ -90,7 +90,9 @@ def self.configure_producer_or_consumer(kafka_config)
       if kafka_config.respond_to?(:bulk_import_id_column) # consumer
         klass.config.merge!(
           bulk_import_id_column: kafka_config.bulk_import_id_column,
-          replace_associations: kafka_config.replace_associations,
+          replace_associations: kafka_config.replace_associations.nil? ?
+                                  Deimos.config.consumers.replace_associations :
+                                  kafka_config.replace_associations,
```

Comment:
Is this purely so we don't have to specify this for each individual consumer?

Reply:
Pretty much